.clang-tidy

```diff
@@ -10,5 +11,8 @@ Checks: >
   -clang-analyzer-osx.*,
   -clang-analyzer-security.insecureAPI.rand,
   -clang-analyzer-unix.Malloc,
+CheckOptions:
+  - { key: readability-identifier-naming.ClassCase,           value: CamelCase }
+  - { key: readability-identifier-naming.FunctionCase,        value: camelBack }
```

Comment:
All the extra spaces here are to accommodate additional options which have longer names.